### PR TITLE
address naming changes for postgresql scaler

### PR DESCRIPTION
**Pull request by dimberman**

### CODE REVIEW

**Comment by dimberman** [comment]
@tomkerkhove can we figure out what other variable name changes we should do?

File: pkg/scalers/postgres_scaler.go [hunk 1]
```diff
@@ -18,26 +18,26 @@ const (
...
+	userName   string
+	password   string
+	host       string
+	port       string
+	query      string
+	dbName     string
+	sslmode    string
 }
 
 var postgresLog = logf.Log.WithName("postgres_scaler")
```
Comment (zroubalik) [inline pkg/scalers/postgres_scaler.go]: please rename this as well to keep things consistent, `postgresLog` `postgresLog`

File: pkg/scalers/postgres_scaler.go [hunk 2]
```diff
@@ -18,26 +18,26 @@ const (
 	defaultPostgresPassword = ""
```
Comment (zroubalik) [inline pkg/scalers/postgres_scaler.go]: please rename this as well to keep things consistent

File: pkg/scalers/postgres_scaler.go [hunk 3]
```diff
@@ -47,31 +47,31 @@ func NewPostgresScaler(resolvedEnv, metadata, authParams map[string]string) (Sca
...
 		return nil, fmt.Errorf("error establishing postgres connection: %s", err)
 	}
-	return &postGRESScaler{
+	return &postgreSQLScaler{
 		metadata:   meta,
 		connection: conn,
 	}, nil
 }
 
-func parsePostgresMetadata(resolvedEnv, metadata, authParams map[string]string) (*postGRESMetadata, error) {
```
Comment (zroubalik) [inline pkg/scalers/postgres_scaler.go]: please rename this as well to keep things consistent. `parsePostgresMetadata`

File: pkg/scalers/postgres_scaler.go [hunk 4]
```diff
@@ -18,26 +18,26 @@ const (
...
 
 var postgresLog = logf.Log.WithName("postgres_scaler")
 
-// NewPostgresScaler creates a new postgres scaler
-func NewPostgresScaler(resolvedEnv, metadata, authParams map[string]string) (Scaler, error) {
+// NewPostgreSQLScaler creates a new postgres scaler
+func NewPostgreSQLScaler(resolvedEnv, metadata, authParams map[string]string) (Scaler, error) {
 	meta, err := parsePostgresMetadata(resolvedEnv, metadata, authParams)
 	if err != nil {
 		return nil, fmt.Errorf("error parsing postgres metadata: %s", err)
```
Comment (zroubalik) [inline pkg/scalers/postgres_scaler.go]: please edit all error and log messages

File: pkg/scalers/postgresql_scaler.go [hunk 5]
```diff
@@ -14,64 +14,64 @@ import (
...
+	password   string
+	host       string
+	port       string
+	query      string
+	dbName     string
+	sslmode    string
 }
 
-var postgresLog = logf.Log.WithName("postgres_scaler")
+var postgreSQLLog = logf.Log.WithName("postgres_scaler")
```
Comment (zroubalik) [inline pkg/scalers/postgresql_scaler.go]: `postgres_scaler`

File: pkg/scalers/postgresql_scaler.go [hunk 6]
```diff
@@ -14,64 +14,64 @@ import (
...
 
-// NewPostgresScaler creates a new postgres scaler
-func NewPostgresScaler(resolvedEnv, metadata, authParams map[string]string) (Scaler, error) {
-	meta, err := parsePostgresMetadata(resolvedEnv, metadata, authParams)
+// NewPostgreSQLScaler creates a new postgres scaler
```
Comment (zroubalik) [inline pkg/scalers/postgresql_scaler.go]: `postgres scaler`

File: pkg/scalers/postgresql_scaler.go [hunk 7]
```diff
@@ -128,49 +128,49 @@ func getConnection(meta *postGRESMetadata) (*sql.DB, error) {
...
 	err = db.Ping()
 	if err != nil {
-		postgresLog.Error(err, fmt.Sprintf("Found error pinging: %s", err))
+		postgreSQLLog.Error(err, fmt.Sprintf("Found error pinging: %s", err))
 		return nil, err
 	}
 	return db, nil
 }
 
 // Close disposes of postgres connections
```
Comment (zroubalik) [inline pkg/scalers/postgresql_scaler.go]: `postgres connections`

File: pkg/scalers/postgresql_scaler.go [hunk 8]
```diff
@@ -128,49 +128,49 @@ func getConnection(meta *postGRESMetadata) (*sql.DB, error) {
...
-func (s *postGRESScaler) Close() error {
+func (s *postgreSQLScaler) Close() error {
 	err := s.connection.Close()
 	if err != nil {
-		postgresLog.Error(err, "Error closing postgres connection")
+		postgreSQLLog.Error(err, "Error closing postgres connection")
```
Comment (zroubalik) [inline pkg/scalers/postgresql_scaler.go]: `postgres connection`

File: pkg/scalers/postgresql_scaler.go [hunk 9]
```diff
@@ -128,49 +128,49 @@ func getConnection(meta *postGRESMetadata) (*sql.DB, error) {
...
 	return messages > 0, nil
 }
 
-func (s *postGRESScaler) getActiveNumber() (int, error) {
+func (s *postgreSQLScaler) getActiveNumber() (int, error) {
 	var id int
 	err := s.connection.QueryRow(s.metadata.query).Scan(&id)
 	if err != nil {
-		postgresLog.Error(err, fmt.Sprintf("could not query PG: %s", err))
+		postgreSQLLog.Error(err, fmt.Sprintf("could not query PG: %s", err))
```
Comment (zroubalik) [inline pkg/scalers/postgresql_scaler.go]: I'd expand the `PG` on these two lines in the messages

File: pkg/scalers/postgresql_scaler.go [hunk 10]
```diff
@@ -183,7 +183,7 @@ func (s *postGRESScaler) GetMetricSpecForScaling() []v2beta1.MetricSpec {
 }
 
 // GetMetrics returns value for a supported metric and an error if there is a problem getting the metric
-func (s *postGRESScaler) GetMetrics(ctx context.Context, metricName string, metricSelector labels.Selector) ([]external_metrics.ExternalMetricValue, error) {
+func (s *postgreSQLScaler) GetMetrics(ctx context.Context, metricName string, metricSelector labels.Selector) ([]external_metrics.ExternalMetricValue, error) {
 	num, err := s.getActiveNumber()
 	if err != nil {
 		return []external_metrics.ExternalMetricValue{}, fmt.Errorf("error inspecting postgres: %s", err)
```
Comment (zroubalik) [inline pkg/scalers/postgresql_scaler.go]: `inspecting postgres:`

**Comment by ahmelsayed** [comment]
Thanks @dimberman and @zroubalik :)

**Pull request merged**